pages/pipelines/wait_step.md.erb

```diff
@@ -46,6 +46,11 @@ _Optional attributes:_
   </tr>
 </table>
 
+<div class="Docs__troubleshooting-note">
+  <h3>When Wait and Block steps are placed one after another</h3>
+  <p>If a wait step is immediately surrounded by a block step, then the wait step would be ignored and only block step would be run.</p>
+</div>
+
```

Comment:
```suggestion
```

Reply:
I think this belonged #1258 ? I've edited that PR to make the info available in both places, so removing here.